fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/JDBCConstants.java

```diff
@@ -119,9 +119,11 @@
     public static final String DEFAULT_TOKEN_SYSTEM = "default-token-system";
 
     /**
-     * Calendar object to use while inserting Timestamp objects into the database.
+     * This Calendar object is not thread-safe! Use CalendarHelper#getCalendarForUTC() instead.
      */
-    public static final Calendar UTC = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+    // DO NOT USE
+    // public static final Calendar UTC = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+    // DO NOT USE
```

Comment:
should we clear this out now?

Reply:
I left there on purpose with those comments as a reminder to not use it.